### PR TITLE
[Multi-tenancy] Update & Query subwallet

### PR DESCRIPTION
**Pull request by baegjae**
This PR enables two features:

- PUT /multitenancy/wallet/{wallet_id} : Update a subwallet 
We can update `wallet_dispatch_type`, `wallet_webhook_urls`, `label`, and `image_url` of the existing subwallet.
- GET /multitenancy/wallets : Query subwallets
We can query with `wallet_name`.

Thanks!
@ianco @TimoGlastra

### CODE REVIEW

**Comment by ianco** [comment]
Looks good!  There is one formatting error:

`aries_cloudagent/multitenant/admin/routes.py:126:91: E501 line too long (91 > 90 characters)`

Are you going to add unit tests?

**Comment by baegjae** [comment]
@ianco the formatting error is fixed.
In fact, unit tests aren't familiar to me. It will take a few weeks because of my schedule.
One solution is that I make another PR for unit tests after merging this.
Thanks!

**Comment by ianco** [comment]
@baegjae thanks for the update!  Re unit tests, the core devs are getting picky about this so probably best to just keep this PR open until you have time to get to the tests.

Thanks!

**Review by ianco — DISMISSED** [review]
Unit tests

**Comment by baegjae** [comment]
@ianco Unit tests are added. Thanks!

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (TimoGlastra) [inline aries_cloudagent/wallet/models/wallet_record.py:112]: Maybe if `wallet.id` is present it should throw an error instead of deleting the property? I think this will help with preventing unexpected behaviour.

**Review by TimoGlastra — COMMENTED** [review]
Thanks @baegjae! I have one small suggestion

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (baegjae) [reply aries_cloudagent/wallet/models/wallet_record.py]: Thanks for your suggestion. It is applied.

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (andrewwhitehead) [inline aries_cloudagent/wallet/models/wallet_record.py:70]: Should likely be `Sequence[str]` not `list`

File: aries_cloudagent/multitenant/admin/routes.py
Comment (andrewwhitehead) [inline aries_cloudagent/multitenant/admin/routes.py:2]: This is a package-local import, it should be with the other relative imports below.

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (andrewwhitehead) [inline aries_cloudagent/wallet/models/wallet_record.py:5]: Same issue with relative imports

File: aries_cloudagent/multitenant/manager.py
Comment (andrewwhitehead) [inline aries_cloudagent/multitenant/manager.py:262]: Why not just `update_settings(new_settings)`?

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (baegjae) [reply aries_cloudagent/wallet/models/wallet_record.py]: Thanks for your reviews. It is fixed.

File: aries_cloudagent/multitenant/admin/routes.py
Comment (baegjae) [reply aries_cloudagent/multitenant/admin/routes.py]: Fixed

File: aries_cloudagent/wallet/models/wallet_record.py
Comment (baegjae) [reply aries_cloudagent/wallet/models/wallet_record.py]: Fixed

File: aries_cloudagent/multitenant/manager.py
Comment (baegjae) [reply aries_cloudagent/multitenant/manager.py]: Oh, it is my mistake. Has been corrected. Thanks!

**Comment by andrewwhitehead** [comment]
@baegjae  FYI there's a conflict in `multitenant/manager.py` due to another PR being merged

**Comment by baegjae** [comment]
@andrewwhitehead I just pushed a commit to resolve the conflict. Thanks!

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**
